app/src/main/java/com/example/homework1/Console.kt

```diff
@@ -1,4 +1,65 @@
 package com.example.homework1
 
 class Console {
+
+    object calculator {
+        fun console() {
+            var onOff = true
+            var result: Double? = null
+            var operation: String? = null
+            var currentResult: Double? = null
+
+
+            while (onOff) {
+                println("Enter the number")
+                val number = (readLine()?.toDouble() ?: 0) as Double
```

Comment:
```suggestion
                val number = readLine()?.toDouble() ?: 0.0
```
либо же кидать ошибку если не правильное число `val number = readLine()?.toDouble() ?: error("not a number)")` либо повторить ввод
каст числа к Double не должен сработать, разные типы